docs/src/pages/getting-started/supported-platforms/supported-platforms.md

```diff
@@ -19,7 +20,7 @@ You can expect Material-UI's components to render without major issues.
 ## Server
 
 Because Material-UI supports server-side rendering, it needs to support the latest, stable releases of [Node.js](https://github.com/nodejs/node).
-Where possible, the [LTS versions that are in maintenance](https://github.com/nodejs/Release#lts-schedule1) are supported. We recommend using **node v10.x** or newer. However we still support **node v8.x**. The support of **node v8.x** will be stopped in Material-UI Version 5.
+Where possible, the [LTS versions that are in maintenance](https://github.com/nodejs/Release#lts-schedule1) are supported.
```

Comment:
Should we explicitly mention node v10? Also, have you considered starting at v12?

Reply:
> Should we explicitly mention node v10?

Definitely.

> Also, have you considered starting at v12?

Will probably be node 12 on release. I think it makes sense to require node 12 even if we release a bit before LTS end. Otherwise we're stuck with it for the lifetime of v5.

The `stable` bundle will change when releasing `@material-ui/core@latest`. It's just a snapshot of the current state of the browserslist query. 

I'll emphasise this more in the PR, add `maintained node versions` as a hint for `node` and also need to add an entry in migration.